config/prod/iwanna-instance.yaml

```diff
@@ -0,0 +1,47 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v10.j2
+stack_name: iwanna-instance
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "joe.smith@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "computevpc"
+  KeyName: "scicomp"
+  AMIId: "ami-0de53d8956e8dcf80"      # Amazon Linux 2 AMI 2.0.20190313
```

Comment:
@zaro0508, @ahayden.  The new Amazon linux 2 AMI on the amazon UI. `ami-0b898040803850657`.  This AMI specified `ami-0de53d8956e8dcf80` has 14 security updates needed which you can do with `sudo yum update`

Reply:
updated to with the latest AMI